schema/me/lot_standings.js

```diff
@@ -0,0 +1,47 @@
+import gravity from '../../lib/loaders/gravity';
+import LotStanding from './lot_standing';
```

Comment:
Not that this saves you any lines :) but maybe a bit more direct (or at the very least _stylish_)